modules/aurora/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.0.0"
+    }
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "aurora_cpu_credits_empty" {
+  alarm_name        = "${var.aurora_db_instance_name}-aurora-cpu-credits-empty"
+  alarm_description = "${var.aurora_db_instance_name} has run out of Aurora CPU credits"
+
+  metric_name = "CPUCreditBalance"
```

Comment:
Mulig vi er mer interesserte i å trigge en alarm hvis vi må betale for ekstra credits (CPUSurplusCreditsCharged). 
På bildet av `CPUSurplusCreditBalance` leser vi at vi kan nedbetale det vi overskyter i samme døgn uten å betale ekstra. 
![image](https://github.com/nsbno/terraform-aws-alarms/assets/33457035/7edc93e0-33ae-4a3d-85b8-276672d3978f)


![image](https://github.com/nsbno/terraform-aws-alarms/assets/33457035/6a60ffd3-d035-4723-b8d0-a6ea9c1a9f68)